scripts/deploy.js

```diff
@@ -179,8 +136,7 @@ async function deployAll(
         tokenFactory,
         create3Deployer,
         tokenManagerDeployer,
-        interchainToken,
-        hyperliquidInterchainToken,
+        ...(isHyperliquidChain ? { hyperliquidInterchainToken: interchainToken } : { interchainToken }),
```

Comment:
```suggestion
        interchainToken,
```
this doesn't make sense, there's only one token contract variable now